front/lib/models/plan.ts

```diff
@@ -196,7 +196,7 @@ Subscription.addHook(
 
 // Plan <> Subscription relationship: attribute "planId" in Subscription
 Plan.hasMany(Subscription, {
-  foreignKey: { name: "workspaceId", allowNull: false },
+  foreignKey: { name: "planId", allowNull: false },
```

Comment:
Will this just update the constraint by itself ?

Reply:
I'm planning to drop the table and re-create them to make sure everything is valid